fuji_server/models/any_of_fair_results_results_items.py

```diff
@@ -2,6 +2,10 @@
 #
 # SPDX-License-Identifier: MIT
 
+# coding: utf-8
+
+from datetime import date, datetime  # noqa: F401
```

Comment:
Is this used here?

Reply:
Don't think so, it's auto-generated by swagger though (and pre-commit didn't seem to mind it).

Reply:
Ah I see, this is indeed different, so let's keep it

Reply:
Ups wrong comment.. as its auto-generated and not complained we better keep it;)